File: CS/ServerSide/Controllers/WebDocumentViewerController.cs
using DevExpress.Web.Mvc.Controllers;
using System.Web.Mvc;

namespace ServerSide.Controllers
{
    public class WebDocumentViewerController : WebDocumentViewerApiControllerBase {
        public override ActionResult Invoke() {
            var result = base.Invoke();
            // Allow cross-domain requests.
            Response.AppendHeader("Access-Control-Allow-Origin", "*");
            return result;
        }
    }
}

File: VB/ServerSide/Controllers/WebDocumentViewerController.vb
Imports DevExpress.Web.Mvc.Controllers
Imports System.Web.Mvc

Namespace ServerSide.Controllers
	Public Class WebDocumentViewerController
		Inherits WebDocumentViewerApiController

		Public Overrides Function Invoke() As ActionResult
			Dim result = MyBase.Invoke()
			' Allow cross-domain requests.
			Response.AppendHeader("Access-Control-Allow-Origin", "*")
			Return result
		End Function
	End Class
End Namespace